src/objects/Line.js

```diff
@@ -107,7 +100,8 @@ class Line extends Object3D {
 
 		const index = geometry.index;
 		const attributes = geometry.attributes;
-		const positionAttribute = attributes.position;
+		const positionAttribute = attributes.position.clone();
```

Comment:
Cloning buffer attributes in `raycast()` is not good for performance.

Reply:
Yes, I also don't think it's appropriate. Do you have any good advice?

Reply:
Instead of transforming the entire buffer attribute, transform the extracted vertices.

Reply:
![image](https://user-images.githubusercontent.com/15603414/177950721-dcf96587-5049-483e-9f5e-861f5a8c56c9.png)
Did you mean this?
`vStart.fromBufferAttribute( positionAttribute, i ).applyMatrix4(matrixWorld) `
 In this way, the performance of each conversion is not high. If there is index data, the same point may be converted multiple times.

Reply:
The performance should be okay. The matrix-vector multiplication is not that expensive.

Reply:
OK, it has been modified. 
If there is no problem in modifying the line raycaster in this way, then modify the raycaster of the point.

Reply:
@Mugen87  
The raycaster of the point has been converted to world space calculation.